src/ui/IconButton/IconButton.jsx

```diff
@@ -1,23 +1,27 @@
 import styled from "styled-components";
 
 export const IconButton = styled.button`
-    width: 32px;
-    height: 32px;
-    padding: 6px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-color: var(--dynamic-button);
-    border-radius: 3px;
-    color: var(--dynamic-text);
-    //todo - move to generic place
-    transition-property: background-color, border-color, box-shadow;
-    transition-duration: 85ms;
-    transition-timing-function: ease;
-    cursor: pointer;
-    border: none;
-    
-    &:hover {
-        background-color: var(--dynamic-button-hovered);
-    }
+  width: 28px;
+  height: 28px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--dynamic-button);
+  border: none;
+  border-radius: 2px;
+  color: var(--dynamic-text);
+  //todo - move to generic place
+  transition-property: background-color, border-color, box-shadow;
+  transition-duration: 85ms;
+  transition-timing-function: ease;
+  cursor: pointer;
+
+  &:hover {
+    background-color: var(--dynamic-button-hovered);
+    padding: 2px;
+  }
+
+  &.plus .details .star {
+    background-color: var(--dynamic-text)
```

Comment:
class not by icon. give a better class name, like a variant to this type of IconButton, just one name and you the places you need.